docker-entrypoint.sh

```diff
@@ -2,6 +2,13 @@
 set -e
 set -o allexport
 
+# Backup current env
+env > /tmp/environment.env
```

Comment:
Ah makes sense.

And this is towards having a common test/dev/production image. It would be good to plan out how this would work, perhaps see how others are doing this.